testing/tests/DevExpress.ui.widgets.dataGrid/keyboardNavigation.tests.js

```diff
@@ -3210,12 +3209,13 @@ QUnit.testInActiveWindow("DataGrid should skip group rows after tab navigation f
         editing: {
             mode: "cell",
             allowUpdating: true
-        },
-        useLegacyKeyboardNavigation: true
+        }
     };
 
     setupModules(this);
 
+    this.keyboardNavigationController._isLegacyNavigation = () => true;
```

Comment:
Usage of the private `_isLegacyNavigation` method instead of the `useLegacyKeyboardNavigation` option looks like a temporary solution to me.